lib/cisco_node_utils/acl.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # RouterAcl - node utility class for RouterAcl config mgmt.
+  class Acl < NodeUtil
+    attr_reader :acl_name, :afi
+
+    # name: name of the router instance
+    # instantiate: true = create router instance
+    def initialize(acl_name, afi, instantiate=true)
+      @acl_name = acl_name
```

Comment:
Please add some error checking for `acl_name` and `afi`.  Also, I personally would enforce `ipv4` and `ipv6` as the only valid `afi` options for the constructor.

```
def initialize(acl_name, afi, instantiate=true)
  fail TypeError unless acl_name.is_a?(String) && afi.is_a?(String)
  fail ArgumentError unless afi == 'ipv4' || afi == 'ipv6'
  ...
end
```